hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/recordcache/RecordCacheImpl.java

```diff
@@ -239,8 +238,9 @@ private void addToInMemoryCache(
         history.nodeIds().add(nodeId);
 
         // Either we add this tx to the main records list if it is a user/preceding transaction, or to the child
-        // transactions list of its parent
-        final var listToAddTo = isChildTx ? history.childRecords() : history.records();
+        // transactions list of its parent.  Note that scheduled transactions are always child transactions, but
+        // never produce child *records*, the scheduled transaction record is treated as a user transaction record.
```

Comment:
...therefore?
```suggestion
        // never produce child *records*, and therefore the scheduled transaction record is treated as a user transaction record.
```

Reply:
it is more correct to say `instead` in this case.  Cause and effect are inverse from what therefore implies.
I'll make the corresponding change.

Reply:
That's fine, I wasn't sure what the connecting phrase should be. Just that it wasn't there and my brain short-circuited